src/data/pokemon-evolutions.ts

```diff
@@ -88,21 +88,88 @@ export class SpeciesEvolutionCondition {
   }
 }
 
-// TODO: Break this up into a MaleCondition and FemaleCondition?
-export class GenderEvolutionCondition extends SpeciesEvolutionCondition {
-  constructor(requiredGender: Gender) {
-    super((p) => p.gender === requiredGender);
-    this.description = "requires gender";
+/**
+ * For Pokemon that require being male, including
+ * Gallade
+ * Mothim
+ * Basculegion
+ * Oinkalogne
+ * Meowstic
+ *
+ * Custom:
+ * Glalie
+ * Huntail
+ */
+export class MaleEvolutionCondition extends SpeciesEvolutionCondition {
+  constructor() {
+    super((p) => p.gender === Gender.MALE);
+    this.description = "requires male";
   }
 }
 
+/**
+ * For Pokemon that require being female, including
+ * Froslass
+ * Wormadam, Vespiquen
+ * Basculegion
+ * Salazzle
+ * Oinkalogne
+ * Meowstic
+ *
+ * Custom:
+ * Gardevoir
+ * Gorebyss
```

Comment:
```suggestion
 * For Pokemon that require being female, including:
 * ```
 * Froslass
 * Wormadam, Vespiquen
 * Basculegion
 * Salazzle
 * Oinkalogne
 * Meowstic
 * ```
 *
 * Custom:
 * ```
 * Gardevoir
 * Gorebyss
 * ```
```